src/Nri/Ui/Carousel/V1.elm

```diff
@@ -0,0 +1,94 @@
+module Nri.Ui.Carousel.V1 exposing (..)
```

Comment:
I think we'll want to limit the exports before publishing

Reply:
+1